cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -2981,33 +3008,31 @@ StatefulSet getKafkaStatefulSet()   {
                 EntityTopicOperator topicOperator = entityOperator.getTopicOperator();
                 EntityUserOperator userOperator = entityOperator.getUserOperator();
 
-                Future<ConfigMap> futToConfigMap;
-
+                Future<ConfigMap> futToLogConfigMap;
```

Comment:
See, this is better then above ;-)

Reply:
But better still if you make the declaration `final`.